masterlist.yaml

```
@@ -23821,3 +23821,11 @@ plugins:
     clean:
       - crc: 0x2358D3DD
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
+
+  - name: 'Grosta.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/59019/' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0x980ECA98
+        util: '[SSEEdit v4.0.4]'
+        itm: 1
```

Same issue as in #2237.

```suggestion
    dirty:
      - <<: *quickClean
        crc: 0x980ECA98
        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 1
```